game/ato/flightplans/flightplanbuildertypes.py

```diff
@@ -33,8 +35,13 @@ class FlightPlanBuilderTypes:
     @staticmethod
     def for_flight(flight: Flight) -> Type[IBuilder[Any, Any]]:
         if flight.flight_type is FlightType.REFUELING:
-            if flight.package.target.is_friendly(flight.squadron.player) or isinstance(
-                flight.package.target, FrontLine
+            target = flight.package.target
+            if target.is_friendly(flight.squadron.player) and isinstance(
+                target, Carrier
```

Comment:
What about LHA? I think there's a parent that captures both of those.

Reply:
We presently do not support refueling at all on LHA in `6.0`.  I'm willing to add it if you think we should.

Reply:
Seems like something we ought to support? I don't have strong feelings.